config/manager/manager.yaml

```diff
@@ -20,6 +20,9 @@ spec:
       containers:
       - command:
         - /manager
+        args:
+          - "--leader-elect"
+          - "--webhook-port=9443"
```

Comment:
Should we also have `- "--metrics-bind-addr=127.0.0.1:8080"` here? https://github.com/kubernetes-sigs/cluster-api/blob/master/docs/book/src/developer/providers/v1alpha3-to-v1alpha4.md#required-kustomize-changes-to-have-a-single-manager-watching-all-namespaces-and-answer-to-webhook-calls

Reply:
I did add that initially as part of the needed change you are pointing above but if you check a bit down in the same document, it is [suggested](https://github.com/kubernetes-sigs/cluster-api/blob/master/docs/book/src/developer/providers/v1alpha3-to-v1alpha4.md#required-kustomize-changes-to-remove-kubeadm-rbac-proxy) to remove it as part of the `Required kustomize changes to remove kubeadm-rbac-proxy`